### PR TITLE
Bookstore: connect to API

### PR DESCRIPTION
**Pull request by David-Kasilia**
### **To Highlight**

- Worked on the postman to make the necessary requests
- Used the axios and thunk to make API calls
- Installed axios and redux thunk
- Fetch the list from the bookstore API when the page loads
- Add and remove works the same way
redux2

### CODE REVIEW

File: src/components/AddBook.js [hunk 1]
```diff
@@ -1,14 +1,17 @@
 import React, { useState } from 'react';
+import axios from 'axios';
 import { useDispatch } from 'react-redux';
 import { v4 as uuidv4 } from 'uuid';
 
 import { addBook } from '../redux/books/books';
 
 const AddBook = () => {
```
Comment (mateo951) [inline src/components/AddBook.js:6]: Please be sure to use createAsyncThunk for creating thunk action creators as it is required in the guidelines 🤓

File: src/components/AddBook.js [hunk 2]
```diff
@@ -17,20 +20,28 @@ const AddBook = () => {
...
+  };
 
-  const addNewBook = () => {
-    if (title !== '' && author !== '') {
-      dispatch(addBook({ id: uuidv4(), title, author }));
-      setBookTitle('');
-      setBookAuthor('');
-    }
+  const addBookToList = async () => {
+    const addNewBook = await axios.post(`${api}`, {
```
Comment (mateo951) [inline src/components/AddBook.js]: All [Bookstore API](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739) calls are done with `fetch` OR `axios` ✔ Well done!

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @David-Kasilia, 

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: src/components/AddBook.js (hunk 2 above)
Comment (David-Kasilia) [reply src/components/AddBook.js]: @mateo951 Hello, I would like to request if am working on both reviews or the first review.Thank you for your review

**Review by mikethreels — APPROVED** [review]
Hi David, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job following the instructions from the previous CR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**